test/pipelines/test_batch_generation.py

```diff
@@ -107,8 +128,8 @@ def test_get_language_api_flows(self):
                 _, kw = make.call_args
                 print('kw:', kw)
                 apis.append(kw['api_name'])
-            assert make.call_count == 2
-            assert apis == ['logging', 'pubsub']
+            assert make.call_count == 3
+            assert apis == ['logging', 'pubsub', 'simplecompute']
```

Comment:
Why is this called "_simple_ compute"?

Reply:
I had it at "simplecompute" because it wasn't the real, full compute API.
Changed it back to "compute" because these are just test resource files and I think other people understand it's not the real thing.